.gitignore

```diff
@@ -11,6 +11,7 @@
 [Oo]bj/
 [Ll]og/
 [Ll]ogs/
+[Bb]uild/
```

Comment:
Just feel like this should be here, given how common the `[Bb]uild` directory is